routes/inertia.php

```diff
@@ -64,4 +74,13 @@
                         ->name('team-invitations.destroy');
         }
     });
+
+    // Socialite...
+    if (Jetstream::hasSocialiteFeatures()) {
+        Route::get('/socialite/{provider}', [SocialiteController::class, 'redirectToProvider'])->name('socialite.redirect');
+        Route::get('/socialite/{provider}/callback', [SocialiteController::class, 'handleProviderCallback'])->name('socialite.callback');
```

Comment:
Maybe `/oauth/{provider}` & `oauth.callback` 🤔, maybe a worthy refinement, maybe not.
Reasoning: `socialite` is the brand of the package, `oauth` is the functionality.

Reply:
@dillingham Thanks for this, just pushed it up! 👍